### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Tammy-Ajoko** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#38] [Outreachy 2022] [answered]
Hello everyone, I'm Tammy from Nigeria, and I've been approved to contribute to the outreachy project, this is new for me, but I found this particular project interesting and I'm here, I need your help and guidance, thank you all .

**Comment by jesutobi** [comment]
i can help you out tammy

**Reply by jesutobi** [reply]
fork and clone the repo, then go through the site and look for ways in which you can contribute,

**Reply by jesutobi** [reply]
> fork and clone the repo, then go through the site and look for ways in which you can contribute,

then create an issue tagging @BatoolMM or @alswajiab  ,they are our mentors

**Reply by Tammy-Ajoko** [reply]
@jesutobi please is there a group chat or community off github where applicants communicate

**Reply by jesutobi** [reply]
yeah, use this link https://github.com/orgs/Open-Science-Community-Saudi-Arabia/discussions

**Comment by BatoolMM** [comment, marked as answer]
Thank you @jesutobi for your help, 

As he mentioned, we have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
